dell/src/main/java/org/apache/iceberg/dell/ecs/EcsOutputFile.java

```diff
@@ -25,18 +25,33 @@
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.OutputFile;
 import org.apache.iceberg.io.PositionOutputStream;
+import org.apache.iceberg.metrics.MetricsContext;
 
 class EcsOutputFile extends BaseEcsFile implements OutputFile {
+
+  /**
+   * @deprecated moving to package-private in 0.15.0
+   */
+  @Deprecated
```

Comment:
Actually, the class is already package-private, so we can just remove it. Sorry I didn't notice that before.

Reply:
I see, I have reverted the changes.